torch/_subclasses/meta_utils.py

```diff
@@ -52,32 +71,42 @@ def assert_eq(a, b):
     assert a == b, f"{a} != {b}"
 
 
-def assert_metadata_eq(assert_eq, m1, m2, *, skip_symbolic=False):
+def assert_metadata_eq(
+    assert_eq,
+    m1: Union[MetaTensorDesc, torch.Tensor],
+    m2: torch.Tensor,
```

Comment:
Why is it we're converting m1 but not m2?

Reply:
m2 is always the produced output meta/fake tensor, it's never a desc